src/commands/generate.ts

```diff
@@ -37,23 +38,23 @@ export default class Generate extends APICommand {
     this.verboseLog('API Key:', this.rawApiClient ? 'Set (hidden)' : 'Not set')
     this.verboseLog('Environment:', this.currentEnvironment)
     this.verboseLog('Base API URL:', process.env.PREFAB_API_URL || 'Default')
-    this.verboseLog('Language:', flags.lang)
+    this.verboseLog('Language:', flags.target)
     this.verboseLog('Output directory:', flags['output-dir'])
 
-    // Get the language from the flag, using lowercase to ensure consistency
-    const langInput = flags.lang?.toLowerCase()
+    // Get the target from the flag, using lowercase to ensure consistency
+    const langInput = flags.target?.toLowerCase()
 
     // Map the input string to the appropriate enum value
     let language: SupportedLanguage
 
-    if (langInput === 'python') {
+    if (langInput === 'python-pydantic') {
       language = SupportedLanguage.Python
-    } else if (langInput === 'react') {
+    } else if (langInput === 'react-ts') {
       language = SupportedLanguage.React
-    } else if (langInput === 'typescript') {
+    } else if (langInput === 'node-ts') {
       language = SupportedLanguage.TypeScript
     } else {
-      throw new Error(`Unsupported language: ${langInput}. Supported languages are: typescript, python`)
+      throw new Error(`Unsupported target: ${langInput}`)
```

Comment:
This else-case is really a developer-only scenario where you've added something to the (new) target options above but haven't added it to this lookup yet. So there's no need to enumerate the list and keep it updated.